docs/4.4/api-reference.md

```diff
@@ -796,14 +798,111 @@ if err := client.DeleteAccessRequest(ctx, accessReqID); err != nil {
 
 ## Certificate Authority
 
-It might be useful to retrieve your [Certificate Authority](architecture/authentication.md#ssh-certificates) through the API if it is rotating frequently.
+Teleport uses SSH Certificates to securely connect servers. To achieve this, the Auth server of a Teleport cluster acts as the [Certificate Authority](architecture/authentication.md#ssh-certificates) (CA), which signs SSH certificates for users and hosts in the cluster. The auth server uses separate CAs for users and hosts.
+
+You may want to use this API to manage your CA if:
+
+- You don't want to automate rotating certificates with `tctl` ([rotating CA with tctl](admin-guide.md#certificate-rotation))
+- You want to set up a custom system for rotating certificates for more security and stability
+- You want to rotate certificates in a trusted cluster without manually fetching the new certs on remote clusters
+
+### Certificate Rotation
+
+To maintain security across your cluster, it is a good idea to set up automatic [certificate rotation](architecture/authentication.md#certificate-rotation). The most fault tolerant, fast, and effective way to rotate the CA is to use a custom solution, but we'll start with the default automated solution and work from there.
```

Comment:
`The most fault tolerant, fast and effective way to rotate` - not sure how an arbitrary custom solution ticks all these points? Has this been explained? Maybe I misinterpreted what a custom solution is in this context?